spec/HttpMethodsSpec.php

```diff
@@ -2,14 +2,16 @@
 
 namespace spec\Http\Client\Utils;
 
+use Http\Client\HttpClient;
 use Http\Client\Utils\HttpMethodsClient;
+use Http\Message\MessageFactory;
 use PhpSpec\ObjectBehavior;
 
 class HttpMethodsSpec extends ObjectBehavior
 {
-    function let()
+    function let(HttpClient $client, MessageFactory $messageFactory)
     {
-        $this->beAnInstanceOf('spec\Http\Client\Utils\HttpMethodsClientStub');
+        $this->beAnInstanceOf('spec\Http\Client\Utils\HttpMethodsClientStub', [$client, $messageFactory]);
```

Comment:
Can you please format this line according to PSR? It is test, and not too long. Still, easier to read that way.